### PR TITLE
Add page for react-native-print component

### PR DESCRIPTION
**Pull request by tritao**
Code_2srfv4FFG7

### CODE REVIEW

File: src/examples/PrintExamplePage.tsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+  const printRemotePDF = async() => {
+    await RNPrint.print({ filePath: 'https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf' })
+  }
+
+  return (
+    <Page
+      title="Print"
+      description="Print documents via the react-native-print module.">
+
```
Comment (chiaramooney) [inline src/examples/PrintExamplePage.tsx:29]: nit: remove extra new line

File: src/examples/PrintExamplePage.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      <View >
+        <Text>For more information about the supported APIs, check out the{' '}
+          <Text style={{color: 'blue'}} onPress={() =>
+             Linking.openURL('https://github.com/christopherdro/react-native-print#usage')}>
+          documentation
+          </Text>
+        .
+        </Text>
+      </View>
```
Comment (chiaramooney) [inline src/examples/PrintExamplePage.tsx:38]: PR just merged which adds a hyperlink section at the bottom of each component page with links to documentation. See the template page as an example. Could you add that section to this page in replacement of this?

**Comment by tritao** [comment]
Updated.

**Review by chiaramooney — APPROVED** [review]
👍 looks great

**Comment by tritao** [comment]
Rebased.

**Comment by tritao** [comment]
CI is failing in what looks like unrelated issues to the PR changes: `There is not enough space on the disk.`.

**Comment by chiaramooney** [comment]
> CI is failing in what looks like unrelated issues to the PR changes: `There is not enough space on the disk.`.

@tritao I spoke with @kmelmon and ideally we'd only like to merge PRs that pass the CI Loop. We ran the CI Loop on main to check if maybe this issue was introduced in a past PR but failed to appear since the CI Loop was not run multiple times, but after testing main is succeeding each time. As a test, could you try fixing the merge conflicts in another of your PRs that are passing the CI Loop right now and seeing if those don't hit this same issue.

**Comment by tritao** [comment]
> > CI is failing in what looks like unrelated issues to the PR changes: `There is not enough space on the disk.`.
> 
> @tritao I spoke with @kmelmon and ideally we'd only like to merge PRs that pass the CI Loop. We ran the CI Loop on main to check if maybe this issue was introduced in a past PR but failed to appear since the CI Loop was not run multiple times, but after testing main is succeeding each time. As a test, could you try fixing the merge conflicts in another of your PRs that are passing the CI Loop right now and seeing if those don't hit this same issue.

Please see https://github.com/microsoft/react-native-gallery/pull/46 which is having the same CI issue.

Github Actions runners only guarantee 14GB of disk space: https://docs.github.com/en/actions/reference/specifications-for-github-hosted-runners#supported-runners-and-hardware-resources

**Comment by chiaramooney** [comment]
@tritao Alright pipeline has been updated and is running in ADO. If you pull in the latest changes from main, you should be good to go.

**Comment by tritao** [comment]
Rebased and passing CI.

**Pull request merged**